File: hw/gudino/u2/hw19 Inheritance/Js/Cage.js
// Cage.js
export class Cage {
    constructor(id, description, type, location) {
        this.id = id;
        this.description = description;
        this.type = type;
        this.location = location;
    }

    toString() {
        return `Cage{id=${this.id}, description=${this.description}, type=${this.type}, location=${this.location}}`;
    }
}

File: hw/gudino/u2/hw19 Inheritance/Python/Cage.py
# Cage.py
class Cage:
    def __init__(self, id, description, type, location):
        self.id = id
        self.description = description
        self.type = type
        self.location = location

    def __str__(self):
        return f"Cage(id={self.id}, description={self.description}, type={self.type}, location={self.location})"
